.github/workflows/core.yml

```diff
@@ -10,15 +10,15 @@ on:
     - cron:  '0 7 * * *'
 
 jobs:
-  core:
+  pod_lib_lint:
     runs-on: macOS-latest
+
+    strategy:
+      matrix:
+        target: [ios, tvos, macos]
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

Comment:
why checkout@v2?

Reply:
See #4764 and https://github.com/actions/checkout

Reply:
That PR doesn't really explain checkout@v2. There's some details in #4752 though, see specifically https://github.com/firebase/firebase-ios-sdk/pull/4752/commits/5ce0b2332cb6c4e39203689cda3e574af6aa39d4.

In short, the major difference between the versions is that v1 does a full clone including all branches, tags, and commits. checkout@v2 does a shallow clone of just one commit from the branch in question. This makes checkouts much faster.